main.go

```diff
@@ -309,6 +394,41 @@ func Exists(path string) bool {
 	return false
 }
 
+/*
+*	This function expands given input globs into matching paths on the server.
+ */
+func ExpandGlobs(cmdExec cmd_exec.CmdExec, paths []string, instance string) []string {
+	var newPaths []string
+	// iterate over each input path
+	for _, v := range paths {
+		// check if path is a glob
+		if strings.ContainsAny(v, "*?[]") {
+			dir := filepath.Dir(v)
+			out, _ := cmdExec.GetFile(appName, dir, instance)
+			body := strings.Split(string(out), "\n")
+			body = body[3 : len(body)-2]
```

Comment:
I think it would probably be worthwhile to add a comment on this line that explains why you sliced this array this way.